.github/workflows/app.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Install poetry
         run: |
-          pip install poetry==1.1.15
+          pip install poetry==1.2.2
```

Comment:
No poetry 1.2.x for this project :D It contains more bugs than you can even imagine, trust me :)

Please change the version and also you forgot to run `poetry update` to update lock file with new version of telegram bot.

Reply:
Do you recommend keeping 1.1.x or to update to 1.3.1 with a new lock file format?

Reply:
Lets keep it 1.1.15 for now.

Reply:
Done. 

Probably I should update `README.md` with a `pip install poetry==1.1.15` as well.

Reply:
There are no need to do that. But probably it will be good to bump minor version. Lets do it next time :)